projects/1.20/assets/the-midnight-lurker/midnightlurker/lang/zh_cn.json

```
@@ -0,0 +1,126 @@
+  "effect.midnightlurker.lurker_angered": "惹怒祟鬼",
+  "effect.midnightlurker.lurker_faker_spawn": "祟鬼假生成",
+  "effect.midnightlurker.lurker_shadow_spawn": "祟鬼影身",
+  "effect.midnightlurkeroverhauled.insanity": "疯狂",
+  "entity.midnightlurker.destroytex": "destroytex1",
+  "entity.midnightlurker.destroytex_2": "destroytex1",
+  "entity.midnightlurker.destroytex_3": "destroytex1",
+  "entity.midnightlurker.destroytex_4": "destroytex1",
+  "entity.midnightlurker.midnight_lurker_aggressive": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_backturned": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_faker": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_faker_aggro": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_faker_watcher": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_hider": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_invisible": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_runaway": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_runtrue": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_seen_angressive": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_shadow": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_shadow_eyes": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_shapeshifter": "化形午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_stalking": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_stare": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_unprovoked": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurker_watcher": "午夜祟鬼",
+  "entity.midnightlurker.midnight_lurkertpose": "午夜祟鬼",
+  "entity.midnightlurker.midnightlurker_ne": "午夜祟鬼",
+  "entity.midnightlurker.spookyambienceentity": "恐怖环境实体",
+  "entity.midnightlurker.void_gateway": "虚空之门",
+  "entity.midnightlurkeroverhauled.destroytex": "destroytex1",
+  "entity.midnightlurkeroverhauled.destroytex_2": "destroytex1",
+  "entity.midnightlurkeroverhauled.destroytex_3": "destroytex1",
+  "entity.midnightlurkeroverhauled.destroytex_4": "destroytex1",
+  "entity.midnightlurkeroverhauled.midnight_lurker": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.midnight_lurker_aggressive": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.midnight_lurker_faker": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.midnight_lurker_faker_aggro": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.midnight_lurker_faker_watcher": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.midnight_lurker_invisible": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.midnight_lurker_ne": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.midnight_lurker_seen_angressive": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.midnight_lurker_stalking": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.midnight_lurkertpose": "午夜祟鬼",
+  "entity.midnightlurkeroverhauled.spookyambienceentity": "恐怖环境实体",
+  "entity.midnightlurkeroverhauled.void_gateway": "虚空之门",
+  "gui.midnightlurker.chasetimerfortesting.label_chase_timer": "追逐计时器：",
+  "gui.midnightlurker.chasetimerfortesting.label_closespawntimer": "关闭/临近生成计时器：",
```

在游戏中的表现如何

---

游戏中没找着，只好采用这种不太靠谱的译法了

---

```suggestion
  "gui.midnightlurker.chasetimerfortesting.label_closespawntimer": "邻近生成计时器：",
```
去翻了源码，应该是这个意思……吧
代码里没有注释，很难顶

---

谢谢翻源码，这我是真搞不来，已采纳